src/lib389/lib389/plugins.py

```diff
@@ -175,6 +175,9 @@ def disable_all_subtrees(self):
 
         self.set('uniqueness-across-all-subtrees', 'off')
 
+    def add_exclude_subtree(self, basedn):
+        self.add('uniqueness-exclude-subtrees', basedn)
```

Comment:
If we are adding a new method, I think we should also add `remove_exclude_subtree` for completeness